topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md

```diff
@@ -0,0 +1,108 @@
+<!--
+|metadata|
+{
+    "fileName": "localizing-igeditors",
+    "controlName": "igEditors",
+    "tags": []
+}
+|metadata|
+-->
+
+# How to localize the Editors
+
+There are two types of localization. First is for the localization resources in the controls. Second is for the regional settings in the controls.
+
+Localization resources for the controls are in Bulgarian, Russian, Japanese, German, Spanish and French languages. These reside in js/modules/i18n (where js is the root folder for the JavaScript files in the Ignite UI program installation path). To get more into using the localization resources you can read the ["Customizing the Localization of Ignite UI Controls"](customizing-the-localization-of-netadvantage-for-jquery-controls.html) topic.
+
+The regional settings, on the other hand, provide localized formats including dates, numbers, and currency symbols for the editors. These reside in the ../js/modules/i18n/regional (where js is the root folder for the JavaScript files in the Ignite UI program installation path).
```

Comment:
provide region specific formats for dates, numbers, as well as currency symbols, floating point symbols, decimal separators, default decimal rounding, etc.